### PR TITLE
Update Markers-and-BC.md

### PR DESCRIPTION
**Pull request by bigfooted**
description of boundary conditions for turbulence and wall models

### CODE REVIEW

File: _docs_v7/Markers-and-BC.md [hunk 1]
```diff
@@ -121,6 +123,40 @@ A marker can be defined as a Farfield boundary by addings its name to the `MARKE
 MARKER_FAR= (farfield)
 ```
 
+## Turbulence Boundary Condition ##
+
+| Solver | Version | 
+| --- | --- |
+| `RANS`, `INC_RANS`, | 7.0.0 |
```
Comment (pcarruscag) [inline _docs_v7/Markers-and-BC.md:130]: 7.3 ?

**Review by pcarruscag — APPROVED** [review]

File: _docs_v7/Markers-and-BC.md (hunk 1 above)
Comment (pcarruscag) [reply _docs_v7/Markers-and-BC.md:130]: I think the idea was "from version X you can do Y"

Comment (pcarruscag) [reply _docs_v7/Markers-and-BC.md:130]: They are not updated.

Comment (bigfooted) [reply _docs_v7/Markers-and-BC.md:130]: ok, clear, I thought it was supposed to be "tested for version 7.0.0"

Comment (bigfooted) [inline _docs_v7/Markers-and-BC.md:130]:
```suggestion
| `RANS`, `INC_RANS`, | 7.3.0 |
```

**Pull request merged**